src/main/kotlin/no/nav/tiltakspenger/arena/service/TiltakepengerPerioderService.kt

```diff
@@ -24,60 +23,44 @@ class TiltakepengerPerioderService(
         private val SECURELOG = KotlinLogging.logger("tjenestekall")
     }
 
-    fun hentTiltakspengerPerioder(ident: String, fom: LocalDate? = null, tom: LocalDate? = null): List<Periode> {
-        val dbRespons: ArenaYtelseResponsDTO =
-            if (Configuration.applicationProfile() == Profile.DEV) {
-                mapArenaYtelserFraDB(arenaSakRepository.hentSakerForFnr(fnr = ident))
-                    .also { LOG.info { "Antall saker fra db : ${it.saker?.size}" } }
-            } else {
-                ArenaYtelseResponsDTO(saker = null, feil = null)
-            }
-
-        val wsRespons: ArenaYtelseResponsDTO =
-            mapArenaYtelser(arenaSoapService.getYtelser(fnr = ident, fom = fom, tom = tom))
-                .filterKunTiltakspenger()
-        LOG.info { "Antall saker fra ws : ${wsRespons.saker?.size}" }
-
+    fun hentTiltakspengerPerioder(
+        ident: String,
+        fom: LocalDate = LocalDate.of(1900, 1, 1),
+        tom: LocalDate = LocalDate.of(2999, 12, 31),
+    ): PeriodeMedVerdier<VedtakDetaljer>? {
         if (Configuration.applicationProfile() == Profile.DEV) {
```

Comment:
Hvorfor gjør denne metoden bare ting i DEV?

Reply:
Av flere grunner. Vi har ikke tilgang til Arena-db i prod ennå, og jeg trenger å teste koden litt i dev. Og vi logger kanskje mer enn vi burde for prod..